### DISCUSSION TITLE
Add multi personal account support

### DISCUSSION THREAD

**Discussion by ClourneySemi** [community/community#56363] [Mobile] [answered]
Now there is only one personal account in the github mobile APP
I want to login 2 personal account and I would switch them  sometime
please

**Comment by eliperkins** [comment]
Hey @ClourneySemi , thanks for this feedback! We hear ya on having multiple GitHub accounts logged into the app and are hard at work to unblock this. Stay tuned!

**Reply by johnlarkin1** [reply]
Hey Eli! Thanks for responding. Is there any update on this?! I also have a personal account and then a work account (that is not actually an Enterprise Account)

**Reply by eliperkins** [reply]
Hey @johnlarkin1 , thanks for the follow-up here!

Starting today, you can beta test our new multi-account functionality in both the [TestFlight beta](https://testflight.apple.com/join/NLskzwi5) and [Play Store beta](https://play.google.com/apps/testing/com.github.android) of GitHub Mobile.

We'll look to polish the feature up ahead of launching the feature publicly in production builds. We'd love your early feedback as someone who has been awaiting this feature!

**Comment by eliperkins** [comment, marked as answer]
Hey @ClourneySemi, @johnlarkin1 and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Comment by Afshi777** [comment]
Link Personal Account in Microsoft

**Comment by Afshi777** [comment]
Link Personal Account in Microsoft
